### DISCUSSION TITLE
Not able to simulate rigorous distillation column

### DISCUSSION THREAD

**Discussion by vishakha-618** [DanWBR/dwsim#589] [General]
Hii @DanWBR 

[Rigorous distillation column.zip](https://github.com/DanWBR/dwsim/files/14023774/Rigorous.distillation.column.zip)
Please see its error.
![Screenshot 2024-01-23 174338](https://github.com/DanWBR/dwsim/assets/156805573/15904e3a-12a3-4799-9575-6f070ab5fdf9)

I have attached file.

**Comment by DanWBR** [comment]
@vishakha-618 the fix for this issue (https://github.com/DanWBR/dwsim/issues/590) will be included in the next update.
